sirepo/sim_data/opal.py

```diff
@@ -34,6 +34,10 @@ def fixup_old_data(cls, data):
             if cmd._type == "filter":
                 cmd.type = cmd.type.upper()
             elif cmd._type == "particlematterinteraction":
+                if cmd.type not in [
+                    t[1] for t in cls.schema().enum.ParticlematterinteractionType
+                ]:
+                    cmd.type = "NOT SELECTED"
```

Comment:
I think this should be `cmd.type = ""`. Again, we want to set it to the value not the display name